### DISCUSSION TITLE
unable to download or update templates whild building up the image

### DISCUSSION THREAD

**Discussion by IGNFIfortrue** [projectdiscovery/nuclei#4774] [Q&A]
Hello guys,

I am trying to build an image contains Nuclei, but after installing Nuclei into it, the templates cannot be installed nor updated, even with -ut. may I ask if there are any ways like curl to download the templates while building up image?

Thanks in advance.

**Comment by olearycrew** [comment]
Could you run `-ut` in the install process? Alternatively, I suppose you could manually download them from `projectdiscovery/nuclei-templates`? 

At run time does the image have access to the internet? What permissions is it running with?

If you could share a Dockerfile that reproduces and/or the error you're getting when running `-ut` it's possible I could help troubleshoot more.

**Reply by IGNFIfortrue** [reply]
Hello [olearycrew](https://github.com/olearycrew),

thank you very much for your answer, I will check my scripts, though -ut still doesnt work in install process when I run nuclei in image, but I think I could manually download from projectdiscovery/nuclei-templates.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
